### DISCUSSION TITLE
Read only custom fields

### DISCUSSION THREAD

**Discussion by samuelsituma** [YetiForceCompany/YetiForceCRM#14272] [How to]
How do I make a custom field in YetiForceCRM read-only?

**Comment by mariuszkrzaczkowski** [comment]
![image](https://user-images.githubusercontent.com/10198654/108812217-08112500-75af-11eb-827a-991bf5c13901.png)

**Comment by samuelsituma** [comment]
Thank you for the response but mine doesn't have that field type option the
version of YetiforceCRM is using is 6.1

**Reply by rskrzypczak** [reply]
change in `/config/Developer.php` false to true and then you see this option

        /** Change of fields visibility */
        public static $CHANGE_VISIBILITY = true;

**Comment by samuelsituma** [comment]
Thank you it's now working perfectly
